changelogs/fragments/2060-zos_mount-skip_fsumf168_from_df.yml

```diff
@@ -0,0 +1,3 @@
+trivial:
```

Comment:
I feel like this one should be labeled as a bugfix

Reply:
you are correct.